events/registration/views.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from django.core.exceptions import PermissionDenied
+from django.views import View
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.http import JsonResponse
+from django.shortcuts import redirect, reverse
+
+from .models import User, RegistrationConfirm
+from .forms import RegistrationForm
+from events import settings
+
+
+CONFIRM_LINK = settings.HOST_NAME + '/api/v1/reg/confirm/'
+
+
+class EmailSender(object):
+    @staticmethod
+    def send_registration_confirm(user):
+        hash_code = RegistrationConfirm.create_confirm(user)
+        subject = 'Confirm registration'
+        message = render_to_string('email.txt', {'link': CONFIRM_LINK + hash_code, 'name': user.first_name})
```

Comment:
Please test if it work with Cyrillic characters or any other not-Latin characters.